spec.html

```diff
@@ -19312,58 +19320,113 @@ <h1>Import Calls</h1>
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-evaluate-import-call" type="abstract operation">
+        <h1>
+          EvaluateImportCall (
+            _specifierExpression_: a Parse Node,
+            optional _optionsExpression_: a Parse Node,
+          ): either a normal completion containing a Promise or a throw completion
+        </h1>
+        <dl class="header"></dl>
         <emu-alg>
           1. Let _referrer_ be GetActiveScriptOrModule().
           1. If _referrer_ is *null*, set _referrer_ to the current Realm Record.
-          1. Let _argRef_ be ? Evaluation of |AssignmentExpression|.
-          1. Let _specifier_ be ? GetValue(_argRef_).
+          1. Let _specifierRef_ be ? Evaluation of _specifierExpression_.
+          1. Let _specifier_ be ? GetValue(_specifierRef_).
+          1. If _optionsExpression_ is present, then
+            1. Let _optionsRef_ be ? Evaluation of _optionsExpression_.
+            1. Let _options_ be ? GetValue(_optionsRef_).
+          1. Else,
+            1. Let _options_ be *undefined*.
           1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
           1. Let _specifierString_ be Completion(ToString(_specifier_)).
           1. IfAbruptRejectPromise(_specifierString_, _promiseCapability_).
-          1. Perform HostLoadImportedModule(_referrer_, _specifierString_, ~empty~, _promiseCapability_).
+          1. Let _attributes_ be a new empty List.
+          1. If _options_ is not *undefined*, then
+            1. If _options_ is not an Object, then
+              1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, « a newly created *TypeError* object »).
+              1. Return _promiseCapability_.[[Promise]].
+            1. Let _attributesObj_ be Completion(Get(_options_, *"with"*)).
+            1. IfAbruptRejectPromise(_attributesObj_, _promiseCapability_).
+            1. [id="step-evaluate-import-call-assert-fallback", deprecated, normative-optional] If the host supports the <emu-xref href="#sec-deprecated-assert-keyword-for-import-attributes">deprecated `assert` keyword for import attributes</emu-xref> and _attributesObj_ is *undefined*, then
+              1. Let _attributesObj_ be Completion(Get(_options_, *"assert"*)).
+              1. IfAbruptRejectPromise(_attributesObj_, _promiseCapability_).
+            1. If _attributesObj_ is not *undefined*, then
+              1. If _attributesObj_ is not an Object, then
+                1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, « a newly created *TypeError* object »).
+                1. Return _promiseCapability_.[[Promise]].
+              1. Let _entries_ be Completion(EnumerableOwnProperties(_attributesObj_, ~key+value~)).
+              1. IfAbruptRejectPromise(_entries_, _promiseCapability_).
+              1. For each element _entry_ of _entries_, do
+                1. Let _key_ be ! <emu-meta suppress-effects="user-code">Get(_entry_, *"0"*)</emu-meta>.
```

Comment:
The `!` [suffices to suppress effects](https://tc39.es/ecmarkup/#effects); you don't need the explicit annotation.

(Or if you do that's a bug; please file on ecmarkup.)

Reply:
It works 👍